### PR TITLE
ViewStage annotation can be defined in a superclass now

### PR DESCRIPTION
**Pull request by metaphore**
Related to #70

### CODE REVIEW

File: mvc/src/main/java/com/github/czyzby/autumn/mvc/component/ui/processor/ViewStageAnnotationProcessor.java [hunk 1]
```diff
@@ -35,7 +35,8 @@ public void processField(final Field field, final ViewStage annotation, final Ob
             throw new GdxRuntimeException("Only Scene2D stages can be annotated with @ViewStage. Found type:"
                     + field.getType() + " in field: " + field + " of component: " + component + ".");
         }
-        final Class<?> controllerClass = field.getDeclaringClass();
+//        final Class<?> controllerClass = field.getDeclaringClass();
```
Comment (czyzby) [inline mvc/src/main/java/com/github/czyzby/autumn/mvc/component/ui/processor/ViewStageAnnotationProcessor.java]: Don't leave commented code in the repository, that's what Git history is for.

**Comment by czyzby** [comment]
Remove the comment, please, and this can be merged.

**Comment by metaphore** [comment]
Oops, sorry about that. 
GWT works fine with this change.

**Comment by czyzby** [comment]
Thanks. I'll upload snapshot soon.

**Pull request merged**